src/main/java/jalau/at18/azul/CenterTileBoard.java

```diff
@@ -0,0 +1,24 @@
+package jalau.at18.azul;
+import java.util.*;
+
+public class CenterTileBoard extends ArrayList<Character> {
+
+    public CenterTileBoard removeTilesOfSpecificColor(char color) {
```

Comment:
Wasn't this removed on a previous PR?

Reply:
Yes it was.  I just realized I created this branch from the other one. I will fix it right now

Reply:
I have fixed the conflicts that came up